backend/src/gpml/handler/review.clj

```diff
@@ -116,7 +116,7 @@
     (list-reviews db reviewer page limit review-status)))
 
 (defmethod ig/init-key ::review-status [_ _]
```

Comment:
i think this should be named differently, perhaps  `::review-status-params` as far as it's injected in `:gpml.handler.review/update-review handler`

Reply:
adding `-params` is not really great but is the way we do in other cases 🤔  ...
because having `:gpml.handler.review-status` integrant key seems to be a handler instead of a bunch of plain data

personally  I don't think it should live in this handler namespace either but at least it'll follow same pattern as i stated before

Reply:
Fair enough, I'll add the `-params` to keep it consistent, for now. We can refactor everything in one shot, later.